src/Platforms/OMPTarget/ompBLAS.cpp

```
@@ -23,6 +23,141 @@ namespace qmcplusplus
 namespace ompBLAS
 {
 
+template<typename T>
+ompBLAS_status gemm_impl(ompBLAS_handle& handle,
+                         const char transa,
+                         const char transb,
+                         const int M,
+                         const int N,
+                         const int K,
+                         const T alpha,
+                         const T* const A,
+                         const int lda,
+                         const T* const B,
+                         const int ldb,
+                         const T beta,
+                         T* const C,
+                         const int ldc)
+{
```

Please throw an error if trans=C.